python/robot-framework/README.md

```diff
@@ -0,0 +1,49 @@
+# Robot Framework Integration Example
+
+This is an example for [Robot Framework](https://robotframework.org/) and [SeleniumLibrary](https://robotframework.org/SeleniumLibrary/) based on SeleniumLibrary's [WebDemo](https://github.com/robotframework/WebDemo) repo.
+
+## Prerequisites
+
+- A device with Python 3+ Support (guide written for unix / macOS)
+- Sauce Labs Account
+
+## Run the demo
+
+- Install & setup Python dependencies (we recommend using a virtual environment)
+
+```sh
+# Create the virtual environment for this project if you haven't already
+python -m venv .venv
+
+# Activate the venv in your shell
+source .venv/bin/activate
+
+# Install dependencies
+pip install -r requirements.txt
+```
+
+- Configure with your Sauce credentials from https://app.saucelabs.com/user-settings
+
+```sh
+export SAUCE_USERNAME=__YOUR_SAUCE_USER_NAME__
+export SAUCE_ACCESS_KEY=__YOUR_SAUCE_ACCESS_KEY__
+# You can change the region to upload to via the SAUCE_REGION env variable.
+# !NOTE! This does NOT change the region the WDIO session is ran against.
```

Comment:
It can be a bit misleading, perhaps? The ondemand url would have to be configured separately of this region variable since that's configured via the SeleniumLibrary when opening the browser. We can remove it from here if this doesn't help.

Reply:
I was mainly surprised by `WDIO` being present here. Then replacing `WDIO` by `SeleniumLibrary` or something similar.

That said, it's also kinda wrong. It can let (me at least) think that you can reach `us-west-1` Webdriver Grid while reaching `eu-central-1` Visual Region (which is not possible).

Reply:
Correct, that will fail which is why I think it would be important to highlight that somewhere that the `SAUCE_REGION` needs to match the ondemand url used in the browser startup. Maybe I should just add a comment there instead (`resource.robot` line 15)?

Reply:
Is `REMOTE_URL` accessible from within the client / binding context ? If so, shouldn't we resolve region from it ?